clang/test/CIR/Tools/cir-translate-target-option/value-invalid.cir

```diff
@@ -0,0 +1,9 @@
+// RUN: not cir-translate --cir-to-llvmir --target foobar --disable-cc-lowering %s 2>&1 | FileCheck %s -check-prefix=INVALID
+
```

Comment:
I don't think we need a new subdir, just call this test `invalid-translate-triple.cir`, or instead just call it `translate-triple.cir` and also add here the test for the warning added below (the split file should help).